drake/thirdParty/tinydir/.travis.yml

```diff
@@ -0,0 +1,24 @@
+sudo: false
```

Comment:
i'm torn as to whether we should have all of the project files from tinydir come into drake.  the copyright file permits modification.  in the extreme, we could keep only the .h file, the COPYING file; it's probably also useful to have the README.md file.  The rest seem like baggage.  But perhaps there are merits to bringing over the entire repo?  What do people think?